src/OrchardCore.Modules/OrchardCore.Alias/Handlers/AliasPartHandler.cs

```diff
@@ -47,6 +48,7 @@ public class AliasPartHandler : ContentPartHandler<AliasPart>
                 templateContext.SetValue("ContentItem", part.ContentItem);
 
                 part.Alias = await _liquidTemplateManager.RenderAsync(pattern, templateContext);
+                part.Apply();
```

Comment:
This should not be necessary ... how are all other parts doing to update their values then?

Reply:
The content part handler implementing the driver mechanism does the same thing: it invokes the content part drivers, then calls Apply. In this case however, it’s not a driver but a content handler directly, so it has to call Apply itself.

Reply:
The content part handler implementing the driver mechanism does the same thing: it invokes the content part drivers, then calls Apply. In this case however, it’s not a driver but a content handler directly, so it has to call Apply itself.

Reply:
Wait, I see what you mean. Let me check how other contentparthandlers (not drivers) behave.

Reply:
There is one other content part handlers that implement the `UpdatedAsync` method, which is `AutoroutePartHandler` - and it does the same thing by calling `part.Apply`:

https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Modules/OrchardCore.Autoroute/Handlers/AutoroutePartHandler.cs?lines=#L118